src/main/java/Main.java

```diff
@@ -0,0 +1,210 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Im Shrek");
+        System.out.println("what are you doing in my swamp!??");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in); //Scanner object named "scanner"
+        ArrayList<Task> tasks = new ArrayList<>(); //Array of "task" object to store the tasks
+
+        //load tasks from file at the start
+        try {
+            tasks = loadTasks();
+        } catch (FileNotFoundException e)
+        {
+            System.out.println("No previous task data found. Starting with an empty list.");
+        }
+
+        String input;
+        while (true) {
+            input = scanner.nextLine();
+
+            try
+            {
+                if (input.equals("bye")) //end loop when user inputs "bye"
+                {
+                    break;
+                }
+                else if (input.equals("list")) //display tasks when user inputs "list"
+                {
+                    System.out.println("____________________________________________________________");
+                    System.out.println(" Here is your list of nonsense, now get out of my swamp:");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println("     " + (i + 1) + "." + tasks.get(i)); //print each line in proper format
+                    }
+                    System.out.println("____________________________________________________________");
+                }
+                else if (input.startsWith("mark")) //mark task and print acknowledge message when user marks a task
+                {
+                    int taskNumber = Integer.parseInt(input.split(" ")[1]) - 1; //get the integer in the text. minus 1 to get correct index in the "tasks" array
+                    tasks.get(taskNumber).markAsDone();
+                    System.out.println("____________________________________________________________");
+                    System.out.println("It’s done. Now, get lost before I toss you out of my swamp!");
+                    System.out.println("     [" + tasks.get(taskNumber).getStatusIcon() + "] " + tasks.get(taskNumber).getDescription());
```

Comment:
Good attempt in trying to separate the long lines of code but indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line.